packages/jsoncpp/jsoncpp.pacscript

```diff
@@ -0,0 +1,53 @@
+pkgname="jsoncpp"
+gives="libjsoncpp25"
+provides=('libjsoncpp-dev' 'libjsoncpp.so')
+conflicts=('libjsoncpp25' 'libjsoncpp-dev' 'libjsoncpp.so')
```

Comment:
.so is a library file name, not a package. We don’t use those in pacstall (at least, not yet)